torch/csrc/autograd/init.cpp

```diff
@@ -432,7 +451,9 @@ static PyMethodDef methods[] = { // NOLINT
   {"set_autocast_cpu_enabled", set_autocast_cpu_enabled, METH_O, nullptr},
   {"is_autocast_cpu_enabled", is_autocast_cpu_enabled, METH_NOARGS, nullptr},
   {"set_autocast_cpu_dtype", set_autocast_cpu_dtype, METH_O, nullptr},
+  {"set_autocast_gpu_dtype", set_autocast_gpu_dtype, METH_O, nullptr},
```

Comment:
Why is this in the autograd code and not in the generic torch. module in `csrc/Module.cpp`?

Reply:
I thought that since there already existed a multitude of functions with similar uses to set_autocast_gpu_dtype in torch/csrc/autograd/init.cpp it would be best to include it there. If there is a strong reason not to do this and move it to csrc/Module.cpp I can very well do that instead, let me know.

Reply:
I agree it is better to keep them together for now indeed.
Just that we might want to clean that up later.

Reply:
Okay, I will keep that in mind for later, thanks for bringing it up.